.github/workflows/image-push.yml

```
@@ -75,8 +75,20 @@ jobs:
           tags: |
             ${{ secrets.AWS_FORGE_ID }}.dkr.ecr.us-west-2.amazonaws.com/anubis:${{ env.PDK_TAG }}
             puppet/pdk:${{ env.PDK_TAG }}
+      -
+        name: Build and push PDK DinD
+        id: docker_build_pdk_dind
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile.dind
+          push: true
+          tags: |
+            ${{ secrets.AWS_FORGE_ID }}.dkr.ecr.us-west-2.amazonaws.com/anubis:dind-${{ env.PDK_TAG }}
+            puppet/pdk:dind-${{ env.PDK_TAG }}
```

There's been a couple of changes since this PR was initially opened, which should be reflected here. (The workflow env var `PDK_TAG` was shortened to `TAG`, and since the anubis image was moved to a dedicated repo, we only need the base PDK image pushed to AWS.)
```suggestion
            puppet/pdk:dind-${{ env.TAG }}
```

---

Update TAG